### PR TITLE
A/B test: additional "request demo" button during HQ signup.

### PR DESCRIPTION
**Pull request by orangejenny**
https://trello.com/c/hxR0twTs/142-a-b-test-new-ui-for-request-demo-during-hq-signup

@gcapalbo / @jmtroth0 

screen shot 2018-08-29 at 10 52 27 pm

### CODE REVIEW

**Review by gcapalbo — APPROVED** [review]

**Review by jmtroth0 — APPROVED** [review]

**Comment by orangejenny** [comment]
added a couple of commits responding to feedback

File: corehq/apps/app_manager/static/app_manager/js/forms/form_designer.js [hunk 1]
```diff
@@ -102,9 +102,9 @@ hqDefine("app_manager/js/forms/form_designer", function() {
         // as the host app. Because we only use requirejs for some parts
         // of the app, appcues gets very confused and throws errors, likely
         // corrupting or invalidating the data in some way. By requiring
-        // appcues to have completed it's init prior to importing requirejs
+        // appcues to have completed its init prior to importing requirejs
```
Comment (jmtroth0) [inline corehq/apps/app_manager/static/app_manager/js/forms/form_designer.js]: nice

File: corehq/apps/registration/static/registration/js/register_new_user.js [hunk 2]
```diff
@@ -3,10 +3,12 @@ $(function () {
     var initial_page_data = hqImport('hqwebapp/js/initial_page_data').get;
 
     // Demo CTA test
-    var kissmetrics = hqImport('analytix/js/kissmetrix');
-    kissmetrics.identifyTraits({'demo UI': (initial_page_data("demo_test") ? "on" : "off")});
-    $("#demo-test-button").click(function() {
-        kissmetrics.track.event("Get Demo button clicked");
+    hqImport("analytix/js/hubspot").then(function() {
+        var kissmetrics = hqImport('analytix/js/kissmetrix');
```
Comment (jmtroth0) [inline corehq/apps/registration/static/registration/js/register_new_user.js]: What does it mean that the demo button requires hubspot? It looks like it shows up even before hubspot has been loaded?

**Review by jmtroth0 — APPROVED** [review]

File: settings.py [hunk 3]
```diff
@@ -950,7 +950,7 @@
 
 _location = lambda x: os.path.join(FILEPATH, x)
 
-IS_SAAS_ENVIRONMENT = SERVER_ENVIRONMENT == 'production'
+IS_SAAS_ENVIRONMENT = True #SERVER_ENVIRONMENT == 'production'
```
[the remark this answers is not in the document]
Comment (orangejenny) [reply settings.py]: thanks stickler for reminding me that this commit shouldn't be released

File: corehq/apps/registration/static/registration/js/register_new_user.js (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/registration/static/registration/js/register_new_user.js]: It's "required" on the html side: both demo buttons are inside of an `{% if ANALYTICS_IDS.HUBSPOT_API_ID %}`

**Review by gcapalbo — APPROVED** [review]

**Pull request merged**